Scripts/NuGet/EdFi.Installer.SandboxAdmin/Install-EdFiOdsSandboxAdmin.psm1

```diff
@@ -13,6 +13,7 @@ step before compiling in C#.
 #>
 
 Import-Module -Force -Scope Global "$PSScriptRoot/Ed-Fi-ODS-Implementation/logistics/scripts/modules/path-resolver.psm1"
+Import-Module -Force -Scope Global "$PSScriptRoot/Ed-Fi-ODS-Implementation/logistics/scripts/modules/utility/hashtable.psm1"
```

Comment:
once we've imported path resolver we should probably use it to resolve the paths to any modules like the nuget and task helpers below.
Also if we bring in this hashtable module here we need to make sure we create a ticket to add it to the EdFi.Installer.AppCommon https://github.com/Ed-Fi-Alliance/Ed-Fi-ODS-Deploy/blob/ce7d1264923cf9a547f96d560b54d5e7c95e1f57/src/EdFi.Installer.AppCommon/EdFi.Installer.AppCommon.nuspec

Reply:
```suggestion
Import-Module -Force -Scope Global $folders.modules.invoke("utility/hashtable.psm1")"
```